Interface.c

```diff
@@ -307,4 +307,69 @@ void executeRedirectCommand(char** args,int argsSize) {
 	
 }
 
+void executePipesCommand(char **args,int argsSize){
+	pid_t pid;
+	int fd[2],returnVal;;
+
+	if((pipe(fd)<0)){
+        	printf("Error creating pipe.\n");
+        	exit(EXIT_FAILURE);
+    	}
+
+	if((pid=fork())<0){
+        	printf("Error forking.\n");
+        	exit(EXIT_FAILURE);    
+    	}
+
+	char *firstCmd = args[0],*firstArg = args[1],*secondCmd = NULL,*secondArg = NULL;
+
+	if (strcmp(args[1],"|") == 0){
+		firstArg = NULL;
+		secondCmd = args[2];
+		if (args[3] != NULL){
+			secondArg = args[3];
+		}
+	} else {
+		secondCmd = args[3];
+		if (args[4] != NULL){
+			secondArg = args[4];
+		}
+	}
```

Comment:
Hard coded pipe detection. What if the command is like this?
`wc -l <somefile>.txt | cat`